src/components/common/TmField.vue

```diff
@@ -175,6 +175,9 @@ export default {
       this.$emit(`input`, formattedValue)
     },
     onChange(...args) {
```

Comment:
2 questions: 

- is this onChange even necessary? 
- why do `number` inputs have to be treated differently?

Reply:
Yes, onChange is needed to update parent's data. I currenly have no answer for second :-(